src/types/any.rs

```diff
@@ -759,9 +759,9 @@ impl PyAny {
     /// use pyo3::types::{PyBool, PyLong};
     ///
     /// Python::with_gil(|py| {
-    ///     let b = PyBool::new(py, true);
+    ///     let b = PyBool::new_bound(py, true);
     ///     assert!(b.is_instance_of::<PyBool>());
-    ///     let any: &PyAny = b.as_ref();
+    ///     let any: &Bound<'_, PyAny> = &*b;
```

Comment:
I wonder if this might be clearer using the new `as_any()`? If you agree, can also change it down below.

```suggestion
    ///     let any: &Bound<'_, PyAny> = b.as_any();
```

Reply:
I agree! I think I started that before the PR merged.